src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -217,8 +222,19 @@ class DropdownAutoCompleteMenu extends React.Component {
     return this.filterItems(items, inputValue).map((item, index) => ({...item, index}));
   };
 
+  getHeight = items => {
+    const {maxHeight, virtualizedHeight, virtualizedLabelHeight} = this.props;
+    const minHeight = virtualizedLabelHeight
+      ? items.reduce(
+          (a, r) => a + (r.groupLabel ? virtualizedLabelHeight : virtualizedHeight),
```

Comment:
Should probably use `!hideGroupLabel` instead of `groupLabel`

Reply:
so it looks like `hideGroupLabel ` is factored in during `filterGroupedItems()` and labels that fall under that criteria are removed from the list:

```js
        // Make sure we don't add the group label to list of items
        // if we try to hide it, otherwise it will render if the list
        // is using virtualized rows (because of fixed row heights)
        if (item.hideGroupLabel) {
          return groupItems;
        }
        return [{...item, groupLabel: true}, ...groupItems];
```

I added this edge case to the 'grouped' storybook example and everything seems fine. But there is a bug where the last element can't be highlighted with the arrow keys when the label is hidden, so I'm looking into that.

Reply:
looks like this bug has always been there, so I'm not gonna let it block this pull. I an address in a separate thing.

Reply:
you can actually see it in action on the project selector if you arrow down super far. it is because react-virtualized alters the order and number of elements that get put into autocomplete.